flocker/node/functional/test_docker.py

```diff
@@ -208,6 +208,23 @@ def started(_):
         d.addCallback(started)
         return d
 
+    def test_list_image_data_cached(self):
+        """
+        ``DockerClient.list`` will only an inspect an image ID once, caching
```

Comment:
The test doesn't actually prove that, though. It just proves the second part, that the information is cached.